diff --git a/build/index.js b/build/index.js
index 0864bdfb05b2ee6a439a4bd320a8a4db5fa0476a..b394c04dfc9f59472c782c658255afe105cba515 100644
--- a/build/index.js
+++ b/build/index.js
@@ -88,7 +88,7 @@ var NotionAPI = class {
         allCollectionInstances,
         async (collectionInstance) => {
           const { collectionId, collectionViewId, spaceId } = collectionInstance;
-          const collectionView = recordMap.collection_view[collectionViewId]?.value;
+          const collectionView = getBlockValue(recordMap.collection_view[collectionViewId]);
           try {
             const collectionData = await this.getCollectionData(
               collectionId,
